src/main/java/SearchEngine.java

```diff
@@ -0,0 +1,72 @@
+import strategies.StrategyChooser;
+import strategies.SearchStrategy;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import java.util.Scanner;
+
+class SearchEngine {
+
+    private final Scanner scanner;
+    private final LineTokenizer lineTokenizer;
+    private final List<String> peopleList;
+    private final StrategyChooser strategyChooser;
+
+    public SearchEngine(FileReader fileReader) throws IOException {
+        peopleList = fileReader.readFile();
```

Comment:
After you read the file you need to do string transformation (remove empty spaces, uppercasing, etc.)
You just do it once then you can remove duplicate logic in other places.